vue-project/src/components/Workflows.vue

```diff
@@ -114,6 +109,23 @@
     }
 
 //// API CALLS ////
+    onMounted( async () => {
+        getWorkflowSteps()
+    })
+
+    const getWorkflowSteps = async () => {
```

Comment:
it might be a good idea to check the status of the HTTP response here and print it to the console if it's non-200

Reply:
Agreed